src/app/pages/component-sidenav/component-sidenav.html

```diff
@@ -1,9 +1,9 @@
 <md-sidenav-container class="docs-component-viewer-sidenav-container">
-  <md-sidenav #sidenav class="docs-component-viewer-sidenav"
+  <md-sidenav aria-label="Side navigation" role="navigation" #sidenav class="docs-component-viewer-sidenav"
```

Comment:
I'm wondering if we should avoid having `<nav>` inside of a `role="navigation"` and instead wrap the entire sidenav content in one `<nav>`